test/e2e/common/node/kubelet_etc_hosts.go

```diff
@@ -112,6 +113,12 @@ func assertManagedStatus(
 
 	retryCount := 0
 	etcHostsContent := ""
+	etcHostsPath := etcHostsPathLinux
+	etcHostsOriginalPath := etcHostsOriginalPathLinux
+	if framework.NodeOSDistroIs("windows") {
+		etcHostsPath = etcHostsPathWindows
+		etcHostsOriginalPath = etcHostsOriginalPathWindows
+	}
```

Comment:
Looks good to me, just one nit: can we somehow avoid duplicating this in 3 places?

Reply:
Done